### PR TITLE
Add sortable to content items, LINK/UNLINK content to list

### PR DESCRIPTION
**Pull request by ahilles107**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | yes
| Fixed tickets             | SWP-661
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Component/Common/Request/RequestParser.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Component\Common\Request;
+
+final class RequestParser
```
Comment (takeit) [inline src/SWP/Component/Common/Request/RequestParser.php]: interface is missing

File: src/SWP/Component/ContentList/Model/ContentListItem.php [hunk 2]
```diff
@@ -55,6 +55,7 @@ class ContentListItem implements ContentListItemInterface
     public function __construct()
     {
         $this->createdAt = new \DateTime();
+        $this->position = -1;
```
Comment (takeit) [inline src/SWP/Component/ContentList/Model/ContentListItem.php]: why `-1` and not `0`?

Comment (ahilles107) [inline src/SWP/Component/ContentList/Model/ContentListItem.php]: hmm... good question - we we create article we want to add it to top of list (first place) or to end (last item)?

Comment (takeit) [inline src/SWP/Component/ContentList/Model/ContentListItem.php]: to the top

**Pull request merged**